.github/workflows/dependency-check.yaml

```diff
@@ -0,0 +1,32 @@
+name: Dependency Check
+
+on:
+  schedule:
+    - cron: "34 14 * * 2"
+  workflow_dispatch:
+
+jobs:
+  dep-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.gradle
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
```

Comment:
Is it necessary to specific a distribution here? We don't seem to need it in https://github.com/mirego/kmp-boilerplate/blob/main/.github/workflows/ci.yaml#L32 👀

Reply:
I opted for a more recent version of the setup-java action, which requires the distribution to be set. We could potentially update the CI workflow and even use the same distribution in the .tools-version file.

Temurin is the recommended distribution of the OpenJDK according to this fine reference:
https://whichjdk.com

Reply:
Great! Thanks for the reference 🙌🏻 Let's update our other github actions and our `.tools-version` then 🙂